### PR TITLE
(blog): Add ios 6.2.0 announcement

### PR DESCRIPTION
**Pull request by NiklasMerz**
### Platforms affected
Blog

### Motivation and Context

Add announcement for upcoming 6.2.0 release

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: www/_posts/2021-02-03-cordova-ios-release-6.2.0.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title:  "Cordova iOS 6.2.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.2.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.2.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
```
Comment (timbru31) [inline www/_posts/2021-02-03-cordova-ios-release-6.2.0.md:15]: Shouldn't we fill in those section?

Comment (NiklasMerz) [reply www/_posts/2021-02-03-cordova-ios-release-6.2.0.md:15]: I put this line at the wrong place.

What are the highlights of this release? Should we mention more. Mostly small stuff as far as I see.

Comment (timbru31) [reply www/_posts/2021-02-03-cordova-ios-release-6.2.0.md:15]: I'm fine if we "label" it as some sort of maintenance release - sometimes updates are not that fancy, but nevertheless important.

**Review by erisu — APPROVED** [review]

File: www/_posts/2021-02-03-cordova-ios-release-6.2.0.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This release contains small bug fixes, dependency updates and resolves some splashscreen issues. Plugin developers can now hook into the `WKURLSchemeHandler ([see more](https://github.com/apache/cordova-ios/pull/1030)).
```
Comment (dpogue) [inline www/_posts/2021-02-03-cordova-ios-release-6.2.0.md:17]:
```suggestion
This release contains small bug fixes, dependency updates and resolves some splashscreen issues. Plugin developers can now hook into the `WKURLSchemeHandler` ([learn more](https://github.com/apache/cordova-ios/pull/1030)).
```

**Pull request merged**